kinoml/features/protein_struct_features.py

```diff
@@ -0,0 +1,185 @@
+"""
+protein.py
+This is a tool to featurize kinase conformational changes through the entire Kinome.
+
+"""
+
+def key_klifs_residues(numbering):
+    """
+    Retrieve a list of PDB residue indices relevant to key kinase conformations mapped via KLIFS.
+
+    Define indices of the residues relevant to a list of 12 collective variables relevant to
+    kinase conformational changes. These variables include: angle between aC and aE helices,
+    the key K-E salt bridge, DFG-Phe conformation (two distances), X-DFG-Phi, X-DFG-Psi,
+    DFG-Asp-Phi, DFG-Asp-Psi, DFG-Phe-Phi, DFG-Phe-Psi, DFG-Phe-Chi1, and the FRET L-S distance.
+    All features are under the current numbering of the structure provided.
+
+    Parameters
+    ----------
+    numbering : list of int
+        numbering[klifs_index] is the residue number for the given PDB file corresponding to KLIFS residue index 'klifs_index'
+
+    Returns
+    -------
+    key_res : list of int
+        Key residue indices
+
+    """
+    if numbering == None:
```

Comment:
When comparing to `True`, `False` and `None`, you should use `is`:

```suggestion
    if numbering is None:
```

This is because all these three objects are _singletons_ (Python guarantees only one instance will ever live in memory), so a memory identity check is enough!